geomstats/geometry/stiefel.py

```diff
@@ -181,21 +250,50 @@ def log(self, point, base_point, max_iter=30):
           Riemannian Logarithm on the Stiefel Manifold under the Canonical
           Metric" SIAM J. Matrix Anal. & Appl., 38(2), 322–342, 2017.
           https://arxiv.org/pdf/1604.05054.pdf
+
+        Parameters
+        ----------
+        point : array-like, shape=[n_samples, n, p]
+            Point in the Stiefel manifold.
+        base_point : array-like, shape=[n_samples, n, p]
+            Point in the Stiefel manifold.
+
+        Returns
+        -------
+        log : array-like, shape=[n_samples, dimension + 1]
+            Tangent vector at the base point equal to the Riemannian logarithm
+            of point at the base point.
         """
-        def normal_component_qr(point, base_point, matrix_m):
-            """Compute QR decomposition of the normal component of a point."""
+        def _normal_component_qr(point, base_point, matrix_m):
+            """Compute the QR decomposition of the normal component of a point.
+
+            Parameters
+            ----------
+            point : array-like, shape=[n_samples, n, p]
+            base_point : array-like, shape=[n_samples, n, p]
+            matrix_m : array-like
+
+            Returns
+            -------
+            matrix_q : array-like
+            matrix_n : array-like
+            """
             matrix_k = point - gs.matmul(base_point, matrix_m)
 
             matrix_q, matrix_n = gs.linalg.qr(matrix_k)
             return matrix_q, matrix_n
 
-        def orthogonal_completion(matrix_m, matrix_n):
-            """Compute the orthogonal matrix completion. TODO: fill in.
+        def _orthogonal_completion(matrix_m, matrix_n):
```

Comment:
I think the `_orthogonal_completion`, `_normal_component_qr` and `procrustes_preprocessing` closures are all too long to be defined in the method body, and should probably be promoted to static methods for readability. Otherwise, their docstrings aren't needed in the first place.

Reply:
Done!